static/js/compose_validate.js

```diff
@@ -62,6 +63,24 @@ export function needs_subscribe_warning(user_id, stream_id) {
     return true;
 }
 
+export function warn_if_pm_mentions_user(full_name, mentioned_user_id) {
+    if (compose_state.get_message_type() !== "private") {
+        return;
+    }
+
+    const recipient = compose_state.private_message_recipient();
+    const emails = util.extract_pm_recipients(recipient);
+    const to_user_ids = people.email_list_to_user_ids_string(emails);
+    if (to_user_ids === mentioned_user_id) {
```

Comment:
This will never be equal since `to_user_ids` is a string and `mentioned_user_id` is an integer.

Reply:
Weird! The warning definitely pops up for me. Anyway, thx, I'll explicitly convert the datatype and resubmit.